### ISSUE TITLE
match `toBe(null)` and possibly `toEqual(null)` in addition to `toBeNull`

### ISSUE THREAD

**Issue by julienw** [testing-library/eslint-plugin-jest-dom#152]
In a few rules (eg: `prefer-document`) `toBeNull()` will be replaced by `toBeInTheDocument()` which is nice.

Is it possible to match `toBe(null)` and `toEqual(null)` as well?

Thanks!

**Comment by benmonro** [comment]
Should be. Care to open a PR?

**Comment by julienw** [comment]
TBH I would love to, but I don't know how to write eslint rules. If you know similar examples I could have a look, but for now I looked at the source code for this rule and I have no idea how to change that...

**Comment by benmonro** [comment]
for sure, it's a great skill to learn and once you do it's actually a ton of fun!  The [docs on eslint](https://eslint.org/docs/developer-guide/working-with-rules) are a good placed to start.  A thing to note, the function names are AST selectors (think CSS selectors, but for code).  that may seem strange, but consider this code:

```js
const foo = { "bar": function() {} };
```
then consider that this function can use the shorthand notation like this:

```js
const foo = { "bar"() {} };
```

an eslint rule uses that pattern (as many of the rules in this plugin do) to declaratively handle various use cases.  So for example, if you wanted a rule to match all CallExpressions you would do 

`{ "CallExpression"(node) { ... } }`

if you wanted a rule to match all call expressions that were named 'foo' (as in `foo()`), you would do this:

`{"CallExpression[callee.name=foo]"(node) { ... }}`

(that might look familiar if you've ever written a CSS selector like `div[role=button]`, that's because it is. :) 
You might be wondering how you can know what all the various types and properties etc, don't worry astexplorer.net has you covered there.  It's a great tool and this entire plugin was written using it.  It's like a dom-inspector for code, and you can play around with rules in real time (among other things).

**Comment by julienw** [comment]
Thanks for the pointers, I think I managed to do something.
